File: Android/app/src/main/java/br/com/desafioandroid/model/Repository.java
package br.com.desafioandroid.model;

public class Repository {
    //@SerializedName("id")
    Integer id;

    String node_id;

    String name;

    String full_name;

    Integer forks;

    Integer stargazers_count;

    boolean priv;

    OwnerRepo owner;

    String description;

    String pulls_url; //29

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNode_id() {
        return node_id;
    }

    public void setNode_id(String node_id) {
        this.node_id = node_id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getFull_name() {
        return full_name;
    }

    public void setFull_name(String full_name) {
        this.full_name = full_name;
    }

    public int getForks() {
        return forks;
    }

    public void setForks(int forks) {
        this.forks = forks;
    }

    public int getStargazers_count() {
        return stargazers_count;
    }

    public void setStargazers_count(int stargazers_count) {
        this.stargazers_count = stargazers_count;
    }

    public boolean isPriv() {
        return priv;
    }

    public void setPriv(boolean priv) {
        this.priv = priv;
    }

    public OwnerRepo getOwner() {
        return owner;
    }

    public void setOwner(OwnerRepo owner) {
        this.owner = owner;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public void setForks(Integer forks) {
        this.forks = forks;
    }

    public void setStargazers_count(Integer stargazers_count) {
        this.stargazers_count = stargazers_count;
    }

    public String getPulls_url() {
        return pulls_url;
    }

    public void setPulls_url(String pulls_url) {
        this.pulls_url = pulls_url;
    }
}

File: app/src/main/java/br/com/renan/desafioandroid/model/data/Repository.kt
package br.com.renan.desafioandroid.model.data

import com.squareup.moshi.Json

data class RepositoryItemsList (
    @field:Json(name = "items") val repositoryItemsList: List<Repository>
)

data class Repository (
    @field:Json(name = "name") val name: String,
    @field:Json(name = "description") val description: String,
    @field:Json(name = "owner") val owner: Owner,
    @field:Json(name = "stargazers_count") val starts: Int,
    @field:Json(name = "forks_count") val forks: Int,
    @field:Json(name = "full_name") val fullName: String
)

data class Owner (
    @field:Json(name = "login") val login: String,
    @field:Json(name = "avatar_url") val avatarUrl: String
)